Exercises/4-range.js

```diff
@@ -1,5 +1,18 @@
 'use strict';
 
-const range = null;
+
+const range = (start, end) => {
+  const arr = [];
```

Comment:
You may allocate array with final length. See solution: https://github.com/HowProgrammingWorks/Reusable/blob/master/Solutions/4-range.js